current/src/urg_time.c

```diff
@@ -20,8 +20,8 @@ void urg_walltime(unsigned long *sec, unsigned long *nsec)
 #else
     struct timeval timeofday;
     gettimeofday(&timeofday,NULL);
-    sec  = timeofday.tv_sec;
-    nsec = timeofday.tv_usec * 1000;
+    *sec  = (unsigned long)timeofday.tv_sec;
```

Comment:
Suggest using a C++-style cast here.

Reply:
done in https://github.com/ros-drivers/urg_c/pull/11/commits/83e68fb8d370a234b9b49018cdbb18c33f6f396f